src/services/objections.api.service.ts

```diff
@@ -0,0 +1,23 @@
+import NewObjection from "../model/new.objection";
+import logger from "../utils/logger";
+
+/**
+ * Create a new objection for the given company.
+ *
+ * @param companyNumber the company number
+ * @param token the bearer security token to use to call the api
+ *
+ * @returns the id of the newly created objection
+ */
+export const createNewObjection = async (companyNumber: string, token: string): Promise<NewObjection> => {
+
+  logger.debug(`Creating a new objection for company number ${companyNumber}`);
+
+  // TODO Call the actual Objections API (via the SDK Node API?) when end-point is implemented. Covered by
+  //      JIRA sub-task BI-4121
+
+  return {
```

Comment:
i'd prefer to return just the id string, that way the client won't have to know about the structure of the object being returned. The model should be in the SDK (when we write it) so that can probably go.

Reply:
ignore that, we do need to return a new Objection. ignore above.

Reply:
After further discussion agreed to just return id as a string. Objection object will come later, when we integrate the call to GET an objection.